WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -182,11 +188,12 @@ private extension BlazeCampaignCreationForm {
                 // Image
                 Image(uiImage: viewModel.image?.image ?? .blazeProductPlaceholder)
                     .resizable()
-                    .aspectRatio(contentMode: .fill)
+                    .aspectRatio(contentMode: .fit)
                     .cornerRadius(Layout.cornerRadius)
 
                 // Tagline
                 Text(viewModel.isLoadingAISuggestions ? "Placeholder tagline" : viewModel.tagline)
+                    .foregroundStyle(.secondary)
```

Comment:
For my learning: What will this modifier do? When should be use this for a Text?

Reply:
Xcode says `foregroundColor` will be deprecated in a future OS version and we should switch to this to reduce the code change we'll need to do in the future. This modifier can apply the same style to a group of text, images, and shapes. You can check out the [documentation](https://developer.apple.com/documentation/swiftui/view/foregroundstyle(_:)) and [this article](https://www.hackingwithswift.com/quick-start/swiftui/how-to-provide-visual-structure-using-foreground-styles) for more information.